registry/storage/linkedblobstore.go

```diff
@@ -263,22 +265,24 @@ func (lbs *linkedBlobStore) Enumerate(ctx context.Context, ingestor func(digest.
 			if err == distribution.ErrBlobUnknown {
 				return nil
 			}
+			retErr = err
 			return err
 		}
 
 		err = ingestor(digest)
 		if err != nil {
+			retErr = err
 			return err
 		}
 
 		return nil
 	})
 
-	if err != nil {
-		return err
+	if retErr != nil {
```

Comment:
I don't see the case where `retErr != err` in this scope

Reply:
If Walk is returning a different error than what `retErr` is set to, can you add a comment explaining that

Reply:
I added a comment.